### DISCUSSION TITLE
This is the coolest plugin I have ever used.

### DISCUSSION THREAD

**Discussion by JMLdev** [Schneegans/Burn-My-Windows#194] [Q&A] [answered]
Thanks for writing such a fun plugin. This makes the OS more enjoyable than any other interface. LOVE it!

**Comment by Schneegans** [comment, marked as answer]
Sorry for not answering earlier - I was on vacation for a couple of days! Thank you so much for the kind words! This means a lot to me!

**Reply by PhilippMDoerner** [reply]
I just want to take some time to emphasize JML's sentiment as well - This extension is just pure fun and basically a must-have on any gnome installation I have. I unironically have chatted with at least half a dozen coworkers at this point when they see an effect through screensharing or the like. It is the perfect example of the kind of cool things that are possible on Linux and when people just go and have fun.

Just thanks a ton for making this!
